challenges/objects-arrays.js

```diff
@@ -85,19 +131,32 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 /* Request 1: .forEach()
 
-The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
+The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and 
+scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
-console.log(animalNames);
+// const animalNames = [];
+// // for (let i = 0; i < zooAnimals.length; i++) {
+// //   animalNames.push(zooAnimals[i].animal_name)
+// // }
+// let animalNamesss = zooAnimals.forEach(function(i){
+//   animalNames.push(zooAnimals[i].animal_name);
```

Comment:
also very close but theres some syntax issue with your function in the for each. maybe look at your arrow functions below and how those are formatted, you could use an arrow function here